spec/factories/access_control_lists.rb

```diff
@@ -2,7 +2,7 @@
 
 FactoryGirl.define do
   factory :access_control_list do
-    user_group
+    association :user_group, factory: :user_group_with_users
```

Comment:
I get they are broken, i just don't want the default factory creating unnecessary DB objects, long term that leads to very slow test suites (slower than what we have already). 

Could you just maybe add another named factory (or a trait) instead of the default that is used everywhere else?

``` ruby
factory :access_control_list_with_user_group do
  association :user_group, factory: :user_group_with_users
end
```